client/src/pages/feedback.tsx

```diff
@@ -116,16 +128,70 @@ const columnHeaders: ColumnDef[] = [
   },
 ];
 
+function FormatMentorQu(
+  mentorAnswers: Answer[],
+  mentorQuestions: Record<string, QuestionState>
+) {
+  const completeAnswers = mentorAnswers
+    .filter((mentorAnswer) => mentorAnswer.status == Status.COMPLETE)
+    .sort((a, b) =>
+      (mentorQuestions[a._id].question?.question || "") >
+      (mentorQuestions[b._id].question?.question || "")
+        ? 1
+        : (mentorQuestions[a._id].question?.question || "") <
+          (mentorQuestions[b._id].question?.question || "")
+        ? -1
+        : 0
+    );
+
+  const incompleteAnswers = mentorAnswers
+    .filter((mentorAnswer) => mentorAnswer.status == Status.INCOMPLETE)
+    .sort((a, b) =>
+      (mentorQuestions[a._id].question?.question || "") >
+      (mentorQuestions[b._id].question?.question || "")
+        ? 1
+        : (mentorQuestions[a._id].question?.question || "") <
+          (mentorQuestions[b._id].question?.question || "")
+        ? -1
+        : 0
+    );
+
+  return completeAnswers.concat(incompleteAnswers);
+}
+
 function FeedbackItem(props: {
+  accessToken?: string;
   feedback: UserQuestion;
   mentorAnswers?: Answer[];
   mentorQuestions: Record<string, QuestionState>;
   onUpdated: () => void;
+  queueList: string[];
+  setQueueList: (queueList: string[]) => void;
 }): JSX.Element {
-  const { feedback, mentorAnswers, mentorQuestions, onUpdated } = props;
+  const {
+    accessToken,
+    feedback,
+    mentorAnswers,
+    mentorQuestions,
+    onUpdated,
+    queueList,
+    setQueueList,
+  } = props;
+  const [currentStatus, setcurrentStatus] = React.useState<Status>(); // for disabling/enabling queue button
+  const [currentID, setCurrentID] = React.useState<string>(); // grab ID of the selected option
```

Comment:
maybe "selectedAnswerId" would be a better name?

Reply:
should currentStatus also be changed to selectedAnswerStatus then?

Reply:
yeah that sounds good!